index.html

```diff
@@ -228,8 +228,8 @@ <h1>Opening hours</h1>
         <div class="footer-right">
             <h1>Get in Touch</h1>
             <p><i class="fa fa-map-marker"></i>Location: Total melen, Yaoundé</p>
-            <p><i class="fa fa-paper-plane"></i>Email: Afrostudio@gmail.com</p>
-            <p></p><i class="fa fa-phone"></i>Phone: +237690920054</p>
+            <p><a href="mailto:Gaelinho@yahoo.fr" target="'_blank"><i class="fa fa-paper-plane"></i>Email: Afrostudio@gmail.com</a></p>
+            <p><a href="tel:6909254"><i class="fa fa-phone"></i>Phone: +237690920054</a></p>
```

Comment:
is it safe to expose your contact on  the website ?

Reply:
yes i think it is safe